lenovo/yoga/6/13ALC6/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, pkgs, ...  }:
+
+{
+  boot.initrd.kernelModules = [ 
+    "amdgpu"
+    "ideapad_laptop"
+  ];
+  services.xserver.videoDrivers = [ "amdgpu" ];
+  hardware.opengl.extraPackages = with pkgs; [
+    rocm-opencl-icd
+    rocm-opencl-runtime
+    amdvlk
+    vaapiVdpau
+    libvdpau-va-gl
+  ];
+  hardware.opengl.extraPackages32 = with pkgs; [
+    driversi686Linux.amdvlk
+  ];
+  hardware.opengl = {
+    driSupport = lib.mkDefault true;
+    driSupport32Bit = lib.mkDefault true;
+  };
+  environment.variables.AMD_VULKAN_ICD = lib.mkDefault "RADV";
+
+  # Latest Kernel
+  boot.kernelPackages = pkgs.linuxPackages_latest;
+
+  hardware.bluetooth.powerOnBoot = lib.mkDefault false;
+  services.power-profiles-daemon.enable = false;
+  services.tlp.enable = lib.mkDefault true;
+  # automatic screen orientation, only works in X11
+  hardware.sensor.iio.enable = true;
+  # energy savings
+  boot.kernelParams = ["mem_sleep_default=deep" "pcie_aspm.policy=powersupersave"];
+  powerManagement.enable = lib.mkDefault true;
+  powerManagement.cpuFreqGovernor = lib.mkDefault "ondemand";
```

Comment:
I think we also had some energy managment options in the repository somewhere.

Reply:
didn't find anything